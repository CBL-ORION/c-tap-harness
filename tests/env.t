#! /bin/sh
#
# Test suite for runtests directory search support.
#
# Copyright 2009 Russ Allbery <rra@stanford.edu>
#
# See LICENSE for licensing terms.

. "$SOURCE/../tap/libtap.sh"

# Total tests.
plan 1

# Be sure that we run from the source directory.
cd "$SOURCE/env"

# Run runtests, overriding the build and source directories.
"$BUILD/../runtests" env.list | sed 's/\(Tests=[0-9]*\),  .*/\1/' > env.result
diff -u env.out env.result 2>&1
status=$?
ok [ $status -eq 0 ]
if [ $status -eq 0 ] ; then
    rm env.result
fi
