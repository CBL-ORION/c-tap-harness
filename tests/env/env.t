#!/bin/sh
. "$SOURCE/../tap/libtap.sh"
real_source="/home/eagle/dvl/c-tap-harness/tests"
real_build="/home/eagle/dvl/c-tap-harness/tests"
plan 2
ok [ "$SOURCE" = "$real_source" ]
ok [ "$BUILD" = "$real_build" ]
