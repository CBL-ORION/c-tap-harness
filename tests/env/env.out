
Running all tests listed in env.list.  If any tests fail, run the failing
test program by hand to see more details.  The test program will have the
same name as the test set but with "-t" or ".t" appended.

env.....ok

All tests successful.
Files=1,  Tests=2
